manifests/p/Prusa3D/PrusaSlicer/2.4/Prusa3D.PrusaSlicer.locale.en-US.yaml

```
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Prusa3D.PrusaSlicer
+PackageVersion: 2.4

```

Needs to match `A`dd or `R`emove `P`rograms, remember to change the folder version name :)
```suggestion
PackageVersion: 2.4.0
```